api/functions/decideStatus.js

```diff
@@ -0,0 +1,38 @@
+import logger from "../utils/logger.js";
+
+import { aggregateUserActivity } from "./aggregateUserActivity.js";
+import { decideScore } from "./decideScore.js";
+
+export const decideStatus = async (configTable, userId, userActivity) => {
+	try {
+		const aggregatedActivity = aggregateUserActivity(userId, userActivity);
+
+		// @todo updated version of decideScore should be replaced when it was mreged
```

Comment:
There's a minor typo in this TODO comment (not critical since it's just a planning note).